modules/navier_stokes/test/tests/finite_volume/ins/solidification/tests

```diff
@@ -0,0 +1,40 @@
+[Tests]
+  design = 'NSComputeLiquidFraction.md NSFVPhaseChangeSource.md NSFVMixtureMaterial.md'
+  issues = '#23357'
+  [solidification_exo_no_advection]
+    type = 'Exodiff'
+    input = 'solidification_no_advection.i'
+    exodiff = 'solidification_no_advection_out.e'
+    requirement = 'The system shall be able to solve solidification without advection using the finite volume method.'
+    ad_indexing_type = 'global'
+    mesh_mode = REPLICATED # because the parsed sideset generator doesn't work with distributed
+  []
+  [solidification_exo_pipe]
+    type = 'Exodiff'
+    input = 'pipe_solidification.i'
+    exodiff = 'pipe_solidification_out.e'
+    requirement = 'The system shall be able to solve solidification with advection in a pipe using the finite volume method.'
+    ad_indexing_type = 'global'
+    mesh_mode = REPLICATED # because the parsed sideset generator doesn't work with distributed
+  []
+  [solidification_enthalpy_balance]
+    type = 'CSVDiff'
+    input = 'pipe_solidification.i'
+    csvdiff = 'pipe_solidification_csv.csv'
+    requirement = 'The system shall be able to conserve energy during solidification.'
+    rel_err = 1.0E-5
+    ad_indexing_type = 'global'
+    mesh_mode = REPLICATED # because the parsed sideset generator doesn't work with distributed
+    recover = false
+  []
+  [solidification_centerline_profile]
+    type = 'CSVDiff'
+    input = 'pipe_solidification.i'
+    csvdiff = 'pipe_solidification_csv_sat_0003.csv'
+    requirement = '(Non regression test) The system must be able to obtain the correct temperature profile over the center line.'
```

Comment:
```suggestion
    requirement = 'The system must be able to obtain the correct temperature profile over the center line.'
```

Reply:
Would also be nice to have a little more context, e.g. "in a solidification problem" ?